README.md

```diff
@@ -75,7 +75,7 @@ roleRef:
 | spec.param.vmUnderTestContainerDiskImage   | VM under test container disk image                                     | False        | Defaults to `quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main`          |
 | spec.param.vmUnderTestTargetNodeName       | Node Name on which the VM under test will be scheduled to              | False        | Assumed to be configured to Nodes that allow DPDK traffic             |
```

Comment:
> Bandwidth units are multiplier * bits/unit of time.

I don't think is accurate. 
This parameter talks about the capacity of the wire, not the speed of packets inside it.
It's like talking about a physical pipe. the dimensions of the pipe are equivalent to the bandwidth (For example, a pipe can be 4 Zoll/Inches in diameter), and Bps is talking about how much fluid (or for our example, let's say "packets" of fluid) can run through it - could be a lot of small packets, or a little big packets. But the dimension is the same.

This parameter is passed directly to the trex_cfg.yaml file (see [manual](https://trex-tgn.cisco.com/trex/doc/trex_manual.html#trex_config_yaml_config_file)), to the `port_bandwidth_gb` param, and is used by trex to know how much memory it needs to allocate.

I would suggest `PortBandwidthGb`

Reply:
I stand corrected, Bandwidth units are in bps